scripts/provisioning/s3_setup

```diff
@@ -177,9 +185,22 @@ configure_openldap()
     /opt/seagate/cortx/s3/install/ldap/setup_ldap.sh --confurl "$confstore_config_url" --defaultpasswd --skipssl
     if [ $? -ne 0 ]
     then
-        echo "ERROR: failed to configure openldap, exiting."
+        echo "ERROR: Failed to configure openldap."
+        echo "ERROR: Please re-run 's3_setup config' again."
```

Comment:
last statement can be followed, we can remove this message

Reply:
done